### PR TITLE
Fix goldenmanga.top

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/AdguardTeam/AdguardFilters/issues/93639

### CODE REVIEW

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
BlockAdblock is always addressed in Base filter. Don't forget to add HTML filter and `$generichide` in content_blocker.txt.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Hi, add the domain to `$$script[wildcard="*eval*babasbmsgx*"][min-length="5000"][max-length="20000"]` too.

**Comment by piquark6046** [comment]
https://github.com/AdguardTeam/AdguardFilters/pull/93669#pullrequestreview-746641710
Why do you want to add this script?

**Comment by Yuki2718** [comment]
> [#93669 (review)](https://github.com/AdguardTeam/AdguardFilters/pull/93669#pullrequestreview-746641710)
> Why do you want to add this script?

We always add this too when it comes to bab. TBH I don't know why exactly, but sometimes scriptlet doesn't work for race condition.

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -140,6 +140,7 @@ infotaxco.com,durtypass.com,dzapk.com,wikihandbk.com,freecourselab.com,onehack.u
 ! BAB 4.2b (`$$script[wildcard="*['length','charCodeAt','fromCharCode'];*(![]+[])*"][min-length="5000"][max-length="27000"]`)
 faucet.dog,dogestar.website,faucet.plus,crypto-faucet.xyz,alobuu.com,omgexploits.com,aloass.com,cryptslice.com,luutruhay.net,taisv.com,aloguy.com,alohdd.com,alogum.com,alobyt.com,aloboi.com,uebnews.online,aloegg.com,alofps.com,pennbookcenter.com,link1s.*,ltc24.com,jagoanssh.com,gainbtc.click,myreadingmanga.info,paidtomoney.com,jpopsingles.eu,pasend.*,freeomovie.to$$script[wildcard="*['length','charCodeAt','fromCharCode'];*(![]+[])*"][min-length="5000"][max-length="27000"]
 ! Old BAB (`$$script[wildcard="*eval*babasbmsgx*"][min-length="5000"][max-length="20000"]`)
+goldenmanga.top$$script[wildcard="*eval*babasbmsgx*"][min-length="5000"][max-length="20000"]
```
Comment (Yuki2718) [inline EnglishFilter/sections/antiadblock.txt:143]: You can just add the domain next to `racevpn.com,`.

Comment (piquark6046) [reply EnglishFilter/sections/antiadblock.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/93669/commits/cf6a61ed4df5c2239e9e830af2c472db763e050c

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
> > [#93669 (review)](https://github.com/AdguardTeam/AdguardFilters/pull/93669#pullrequestreview-746641710)
> > Why do you want to add this script?
> 
> We always add this too when it comes to bab. TBH I don't know why exactly, but sometimes scriptlet doesn't work for race condition.
> @zloyden

This rule existed before scriptlets. So, the main for this.
1. Sometimes race condition.
2. If it's possible to remove ad script, we have to do it.

**Pull request merged**